Create finance.md

benhylau opened the pull request.

benhylau commented:
@hyphacoop/finance-wg should we add banking and billing info to this? Like bank account number and wire transfer details, etc.

patcon commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+# Financial Accounts and Practices
+
+_Note: Note all of the following have been implemented yet, the Finance WG is working on it_
```
```suggestion
_Note: Not all of the following have been implemented yet, the Finance WG is working on it_
```

dcwalk replied:
"Has" vs. have

patcon commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+
+This document outlines finance and accounting practices of Hypha Worker Co-operative, including:
+
+- Listing of accounts at our financial institutions
```
Should we list this in https://link.hypha.coop/inventory too? (They're heavier old-world platforms, but in some ways Desjardins feels no different than Google Docs or Twitter or HubSpot...)

Happy to link out to this doc for specific details though! Should be here, just like we wouldn't keep bitcoin addresses in our hackmd ;)

benhylau replied:
Since CRA is on there I think Desjardins (and Transferwise + QB when we have them) should be on there too. I have added Desjardins and linked.

patcon commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+
+Our main financial institution is **Desjardins** and our main funds are stored in a business chequing account, where all outgoing transactions require 2 signatures from Members of the Finance WG.
+
+Hypha is expected to conduct business with clients from other countries and is ready to [accept funds in CAD, USD, and EUR](https://transferwise.com/help-smart/11/getting-started/2571907/what-currencies-do-you-support). We use **TransferWise** to minimize transaction and currency exchange fees. Our Transferwise account has local accounts in Canada, United States, and Europe, so we can bill client ...
```
Added mention of this responsibility to https://github.com/hyphacoop/organizing/issues/121 where finance role is being fleshed out.

patcon commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+
+Hypha charges clients on a sliding scale, and projects may be billed hourly, weekly, monthly, or per project, depending on the specifics of each project. For all projects, regardless of how a customer is charged, Members log their hours using **Clockify**. For clients and billing models that require timesheets, Clockify is used to generate these timesheets to calculate the amount to invoice each  ...
+
+### Generating timesheets
```
Once we approve general flow, can we count screenshots as part of done-done, before we merge this?

dcwalk replied:
See my larger comment that maybe we merge this earlier and do refinement over time. I think a monolithic long standing PR feels harder to merge over time

patcon commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+
+The general guidance is that the client is responsible for paying applicable GST and transaction costs charged by their financial institution, and Hypha absorbs the cost of receiving the money (e.g. cost of receiving a wire).
+
+Invoices (and timesheets, if requested) are sent to clients and are payable in CAD, USD, or EUR. Invoices are due 14 days after the invoice date, after which a 5% late fee will be charged for each additional week.
```
I've voiced this a few times, but I've been told in a few places that it's better practice to inflate (service fee or whatever), and offer a discount for early payment. My feeling is that there's zero cost to doing it this way, and the framing is inherently friendlier when the inevitable late-payments happen -- we are less likely to hear "please, I'm only 10 hours late, don't ding me", because it's framed as a favour we're offering them, not a penalty.

benhylau replied:
I have read your **discount for early payment** proposal before and you did mention it a couple time earlier, but I feel it doesn't fit well into the way we bill at the moment. So far our work has been on a negotiated hourly rate / fixed project budget, it seems weird to on the bill inflate that amount and then discount it.

Also agree with @YurkoWasHere's points about not actually exercising the late fee, and having cumulative over time.

@patcon if there are particular projects where you quoting / invoicing with the discount structure is suitable, I'd just do that on a project basis, but I don't feel it's a suitable org-wide strategy.

dcwalk replied:
I agree overarchingly with Yurko's points about connection b/w discount and late payment. I'm used to inflation/discounts being treated as optics separate from late payment (which I think we should have flexible policies on until we develop a sense of what works for us).

My general take is to be upfront/fixed with our costs (relying on community pricing models) over a payment discounts. Previous company I worked at with many small contracts b2b never had a practice discounting, I can say from experience that any changes to pricing structure were met with confusion and frustration by usually the limited set of clients who always took more touch to deal with.

All that said. I think the proposed late policy from @benhylau has pretty tight term+high percentage that I would soften (I'm not used to things kicking in until _overdue_ by a set time after a peroid). I think also setting up a payment schedule in contracts is probably the best way to handle getting paid over fees as a mechanism

patcon replied:
Could we agree to just make this a guideline?

If I'm running a small client through the co-op, I just want to be able to choose my own language and framing that suits my sense of human-centred relationships. I like a gift more than a penalty. I don't mind if others don't want to do late fees on their projects or larger ones, I just would prefer this not pushed on my own initiatives.

Is that possible?

patcon replied:
Speaking of which, thoughts on using IEEE spec language throughout our policies? SHALL MUST SHOULD etc

patcon replied:
```suggestion
Invoices (and timesheets, if requested) SHOULD be sent to clients and SHALL be payable in CAD, USD, or EUR. Invoices SHOULD be due 14 days after the invoice date, after which a 5% late fee SHOULD be charged for each additional week.
```

patcon replied:
> I think we should have flexible policies on until we develop a sense of what works for us

:+1: or also, i'd be fine to use IETF lingo as that flexibility.

> I can say from experience that any changes to pricing structure were met with confusion and frustration

:+1: agreed. imho changes to running contracts SHOULD be avoided, and we SHOULD try to preserve agreements a client started with. Would rather be able to handle variations than assume we'll decide/maintain globally consistent terms

> pretty tight term+high percentage that I would soften

:+1: but no block. pls someone suggest better numbers from experience. otherwise, happy to leave ben's. if it's too high, we'll hear about it and can adapt.

> I'm not used to things kicking in until overdue by a set time after a peroid

:-1: In favour of ben's single date over the multiple implied here: "pay by" date and then "overdue" date. KISS

> setting up a payment schedule in contracts is probably the best way to handle getting paid over fees as a mechanism

can you offer a clear example of the distinction?

benhylau replied:
I have left the late fee % non-specific and changed to "_may_ be charged" to address concerns from @dcwalk and @hyphacoop/infra-ops-wg 

I also don't want two dates of payment. After "pay by" it is immediately "overdue".

re: IETF lingo, I am open to but feel it should be adopted across the broadly handbook if we decide to do so, so let's leave for future PRs.

>@patcon if there are particular projects where you quoting / invoicing with the discount structure is suitable, I'd just do that on a project basis, but I don't feel it's a suitable org-wide strategy.

I want to warn against diverging practices in general, as it could create more administrative burden we haven't considered, but we should discuss this when it comes time to make a proposal.

patcon replied:
Just wanted to note that I'm not +1 on the attempt to "avoid divergent practices". I feel quite differently -- that we should usually protect space for divergent practices to exist for a time, and only converge later.

But +1 to the rest! Thanks!

patcon commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+
+## Member payroll
+
+Member salaries are paid monthly and vary month-to-month depending on their billable hours of work. At the end of each month, each Member's entitled pay is calculated based on the projects and roles they participate in. After withholding taxes and other applicable deductions (e.g. CPP and EI), the net amount is electronically tranferred to the Member's bank account.
```
Sorry, did we lock in monthly? I missed this, but would have advocated for biweekly.

dcwalk replied:
yes I also don't recollect this being announced.

Also the above does not use terminology inline with any of the requirements for paying (e.g., pay period, salary vs. wage) and I think we should follow that.

Have you reviewed the ESA around wages and pay? Might be helpful to link to direct sections, for instance we have to pay people within a set date of end of pay period, etc...

dcwalk replied:
We should use the correct terminology when we are discussing all of these things.

Are we paying salaries? or wages?
We are implying a monthly pay period, that follows the calender month, can we be explicit?

etc...

why wouldn't use the proper terms?

benhylau replied:
I changed the wage language and left pay period ambiguous, let me know if this looks okay.

benhylau replied:
Can @dcwalk or @patcon resolve this if looks ok?

patcon commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+
+Hypha has the following accounts with the Canada Revenue Agency (CRA):
+
+- **RC** for corporate income tax, which is reported and remitted each year
```
Can we call them the human-friendly versions, and put the weird system codes in brackets?
https://www.canada.ca/en/revenue-agency/services/tax/businesses/topics/changes-your-business/adding-accounts-your-business-number-bn.html

```suggestion
- **Corporate Income Tax** (RC), which is reported and remitted each year
```

patcon commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+
+### Reporting and remitting taxes
+
+Hypha has the following accounts with the Canada Revenue Agency (CRA):
```
Mind if we stick we their language from: https://www.canada.ca/en/revenue-agency/services/tax/businesses/topics/changes-your-business/adding-accounts-your-business-number-bn.html

I suspect the term "account" is going to be very overloaded in financial convos :)

```suggestion
Hypha has the following _program accounts_ with the Canada Revenue Agency (CRA):
```

YurkoWasHere commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+
+## Organizational structure
+
+Hypha Worker Co-operative is a worker co-operative incorporated in Ontario, Canada. The business has seven Founding Members, four of which are First Directors with signing authority. As a non-share-capital worker co-operative, we are a non-profit organization, and every Member holds both a Worker and an Owner relationship with the business.
```
```suggestion
Hypha Worker Co-operative is a worker co-operative incorporated in Ontario, Canada. The business has seven Founding Members, of which three of the four First Directors have with signing authority. As a non-share-capital worker co-operative, we are a non-profit organization, and every Member holds both a Worker and an Owner relationship with the business.
```

dcwalk replied:
what is a "worker" relationship?

Could you say: every Member is both an employee and co-owner of the business

patcon commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+- **RC** for corporate income tax, which is reported and remitted each year
+- **RP** for payroll deduction, which is reported and remitted each month
+
+As a non-profit organization, our co-operative should have tax-exempt status and pay no corporate income tax. We do not have to collect GST/HST from clients and get GST/HST refunds for our puchases.
```
This isn't my understanding. We don't need to collect HST for mission work (like education or workshops) but **we _do_ need to collect for traditional web dev work for traditional clients**. (Though our sub-$30,000 general threshold for HST means we can avoid it for a bit.).

This information is from Matt @ 4th Pig, who says their construction company 100% has to collect HST for home builds, but don't need to collect HST for some grant-supported work, or for education offerings they make.

I feel like we should default to collecting HST (after we're over over $30,000 threshold) until we have more certainty about what the "mission-based" carve-outs are. This might require convo with Fourth Pig.

dcwalk replied:
Yes this maps to my understanding we'll need to collect HST

patcon commented:
Thanks Ben! This looks rad, and I totally appreciate all your time on it :)

My soft preference is to merge/finalize this until we have someone in the finance role (who's actually thinking through a list of agreed responsibilities), and when we've actually exercised these processes to some extent.

benhylau commented:
Updated the process for timesheet generation to be less burdening.

Is this good for merge?

benhylau commented:
> Thanks Ben! This looks rad, and I totally appreciate all your time on it :)
> 
> My soft preference is to merge/finalize this until we have someone in the finance role (who's actually thinking through a list of agreed responsibilities), and when we've actually exercised these processes to some extent.

@patcon you mean merge or not merge this?

benhylau commented:
> @hyphacoop/finance-wg should we add banking and billing info to this? Like bank account number and wire transfer details, etc.

Would like some input on this @patcon @dcwalk @YurkoWasHere

patcon commented:
> @patcon you mean merge or not merge this?

Sorry, mild pref is to leave this PR open as living document until we've defined a finance person. Closing/finalizing this should be their thing imho

dcwalk commented on finance.md:
```diff
@@ -0,0 +1,70 @@
+- Process for payroll and expense reimbursement to Members
+- Tools and procedures for bookkeeping, accounting, and tax remittances
+
+## Organizational structure
```
This seems like it should be moved into another section from finance?

patcon replied:
This was moved to onboarding. That work?

dcwalk commented:
@benhylau I'm not sure what is left for me to look at here, but took a quick look through.

I think there are a couple steps maybe to land this?
- rebase on master & add into gitbook summary/toc
- (maybe) add it to like an "In progress" section for processes we have agreed to but aren't exactly implemented
- **Stretch** Move/integrate content into relevant sections handbook, in particular with an eye to making things like **individual role and working group responsibilities and deadlines** clear and collected in one place. e.g., 
  - move some chunk of the processes you've described into guides page (i would imagine still grouped as finances)
  - add a role that clearly documents the responsibilities for payroll person inline with cultivator to that page
  - add a "working group resp." sort of section which just like roles bullet point lists the responsibilities for that wg? (these could also take from our initial descriptions which are currently only on loomio.
  - At some point I think we'll also want something like calendar deadlines (maybe a page but probably just added to our collective calendar) for some of the things you mention.

I am happy to help with the stretch stuff, but maybe that happens best in a few smaller PRs

dcwalk commented on finance.md:
```diff
@@ -0,0 +1,69 @@
+
+## Financial accounts
+
+Our main financial institution is **Desjardins** and our main funds are stored in a business chequing account, where all outgoing transactions require 2 signatures from Members of the Finance WG.
```
Fine with setting a threshold

patcon replied:
Action moved here: https://github.com/hyphacoop/organizing/issues/147

patcon commented:
In future docs, can we use newlines per sentence or even per sentence _fragment_ (e.g., breaking on commas) instead of per paragraph?

This feels like it would make review and discussion easier. Some paragraphs have multiple things to respond to

patcon commented:
> (maybe) add it to like an "In progress" section for processes we have agreed to but aren't exactly implemented

Really love this idea. This feels so expansive that I'd love to just merge it in as best-effort, with understanding that it's loose, and then open discussions on smaller points to refine it over time :) We can promote it to a more "official" section once we've lived with it a bit?

For that work to be migrating it into roles, that also sounds like a great idea. Don't want to make this languish forever in a PR! 😅

benhylau commented:
Changes I made:

- Changed to one-sentence-per-line practice
- Merged master and added to TOC
- Using 🚧 to mark page as wip until official guideline on how to handle unimplemented practices
- Move **stretch** to https://github.com/hyphacoop/organizing/issues/121#issuecomment-554994755 and https://github.com/hyphacoop/handbook/issues/32

I would like some approvals to merge this soon rather than keep it hanging here.

patcon commented on onboarding.md:
```diff
@@ -4,13 +4,19 @@ Welcome to Hypha! This page is supposed to help those getting situated in our or
+## Organizational Structure
+
+Hypha Worker Co-operative Inc. is a worker co-operative incorporated in Ontario, Canada.
+The business has seven Founding Members, of which three of the four First Directors have signing authority.
```
```suggestion
The business has seven Founding Members, of which three of the four First Directors have financial signing authority.
```

patcon replied:
I can still sign resolutions and all other legal documents, like any other director :)

benhylau replied:
Moved the section and no longer refers to bank signing.

```suggestion
The business has seven Founding Members, of which four Directors have signing authority.
```

patcon commented on onboarding.md:
```diff
@@ -4,13 +4,19 @@ Welcome to Hypha! This page is supposed to help those getting situated in our or
+
+Hypha Worker Co-operative Inc. is a worker co-operative incorporated in Ontario, Canada.
+The business has seven Founding Members, of which three of the four First Directors have signing authority.
+As a non-share-capital worker co-operative, we are a non-profit organization, and every Member is both an employee and co-owner of the business.
```
```suggestion
As a non-share-capital worker co-operative, we are a non-profit organization, and every Member is both a worker and co-owner of the business.
```

As I understood "employee" has legal meaning, and we're actually just owners right now (we don't have higher threshold of employee rights)?

Correct me if wrong. cc @dcwalk

dcwalk replied:
We are working with the interpretation (documented in business plan) that we are either permanent or *founding* employees. So the original version is correct.

patcon commented on SUMMARY.md:
```diff
@@ -8,5 +8,6 @@
   - [Meetings](meetings.md)
   - [Roles](roles.md)
   - [Working Open](working-open.md)
+  - [Finance](finance.md)
```
```suggestion
  - [Finance 🚧](finance.md)
```

benhylau replied:
Don't think it's necessary to do this in two places.

patcon replied:
I'd like to know from contents what is under construction, assuming this is a practice we widely adopt. If no objection, this small thing is part of what is letting me be ok with a documentation practice that otherwise feels off-base to me. Please consider.

dcwalk replied:
This was added for the coordination practices pr, #29  so mirroring here.

benhylau commented:
It seems the only major concern is https://github.com/hyphacoop/handbook/pull/22#discussion_r347634315 and other changes that were made to address what @dcwalk flagged. So I'd like @dcwalk to merge if she thinks it's good to go.

dcwalk commented:
Reviewed the remaining comments and this seems ready to go in as an In progress/under construction page
